### PR TITLE
refactor: prettify chunk management test

### PR DESCRIPTION
**Pull request by matklad**
Adding an explicit "test fixture" struct makes call-sites more readable (named args!) and makes making further changes easier.

No particular grand plan here, just figured this small change to be a marginal improvement!

### CODE REVIEW

File: integration-tests/src/tests/client/chunks_management.rs [hunk 1]
```diff
@@ -373,101 +371,83 @@ fn chunks_recovered_from_others() {
...
-                false,
-                &["test1", "cop1"],
-                true,
-                2 * CHUNK_REQUEST_SWITCH_TO_FULL_FETCH_MS,
-            );
-        })
-    });
+    Test {
+        validator_groups: 4,
+        chunk_only_producers: true,
```
Comment (matklad) [inline integration-tests/src/tests/client/chunks_management.rs:447]: Huh, this is an unintended bugfix I think, this would should be `true`

**Comment by robin-near** [comment]
Ahh this is so much better. Thanks for the refactoring and fix!

File: integration-tests/src/tests/client/chunks_management.rs (hunk 1 above)
Comment (mzhangmzz) [reply integration-tests/src/tests/client/chunks_management.rs:447]: I'm not sure if chunk_only_producers would be true here. The naming for the rest of the tests seem to indicate that if chunk_only_producers is true, the name is the tests would end with _cop

Comment (matklad) [reply integration-tests/src/tests/client/chunks_management.rs:447]: And this test does end with `_cop`

Comment (matklad) [reply integration-tests/src/tests/client/chunks_management.rs:447]: Ie, it was wrongly `false` before.

Comment (mzhangmzz) [reply integration-tests/src/tests/client/chunks_management.rs:447]: ah sorry I must have misread something

**Review by mzhangmzz — APPROVED** [review]

**Pull request merged**